### DISCUSSION TITLE
Elección de proyecto

### DISCUSSION THREAD

**Discussion by jadercaro** [rramosp/ai4eng.v1#281] [00 - General]
Quiero saber si mi dataset elegido es adecuado, ya hemos hecho varios avances en el tratamiento del dataset (data cleaning) adjunto el link del dataset y del github con los avances:

link dataset: https://www.kaggle.com/datasets/parisrohan/credit-score-classification
link github: https://github.com/jadercaro/ClasificacionPuntajesCredito

**Comment by Felipe-RA** [comment]
Saludos @jadercaro 

Entiendo que tu tarea predictiva es, según lo que tienes en el github, _" predecir la clasificación del score crediticio en solicitudes de crédito bancario"_

Lamentablemente tu dataset no cumple el requisito de dimensionalidad, pues requiere un mínimo de 30 columnas. 

Debes buscar otro

Animos!

**Reply by jadercaro** [reply]
@Felipe-RA Buenas tardes, encontramos esta base de datos que pertenece a una competición y cumple con los requerimientos minimos, por favor confirmenos si el dataset es adecuado.
link kaggle: https://www.kaggle.com/competitions/playground-series-s3e20

gracias

**Reply by Felipe-RA** [reply]
Saludos @jadercaro 

Esta competición cumple con los requisitos, pero lamentablemente ya la han elegido múltiples equipos y por lo tanto hemos vetado nuevos registros.

PD: Lamento la tardanza en responderte, como lo envíaste el viernes por la tarde no vi este mensaje hasta hoy en la mañana.,

**Reply by jadercaro** [reply]
@Felipe-RA  
Ok, no conocíamos esa condición adicional.

Encontramos las siguientes bases de datos:
1) https://www.kaggle.com/competitions/sberbank-russian-housing-market/
2)https://www.kaggle.com/competitions/porto-seguro-safe-driver-prediction
3)https://www.kaggle.com/competitions/amex-default-prediction
4)https://www.kaggle.com/competitions/lish-moa

quedamos atentos, gracias.

**Reply by Felipe-RA** [reply]
No te preocupes! @jadercaro.

Te los desgloso 1 a 1:

1.  Cumple inicialmente, pero asegurate de que tenga suficientes datos faltantes (sino, simulalos). También lee un poco el foro de discusiones de esta competencia, te va servir mucho si la decides tomar.
2. Cumple. Recomiendo que lean muy bien la descripción para entender la naturaleza de los datos, especialmente para entender cómo se representan los datos faltantes aquí, pues es particular.
3. Cumple inicialmente, asegurate de si tiene los datos faltantes, y tiene algunas observaciones. La primera es que los datos estan anonimizados, por lo que no puedes saber que es cada columna y se pierde interpretabilidad. La segunda es que es muy pesado, tendrian que aplicar alguna forma **valida** de reducirlo.
4. Cumple inicialmente, pero asegurate si tiene suficientes datos faltantes y sino simulalos. Relaciona tablas de muchos archivos, un reto interesante y un problema muy interesante también.  Opinión personal, es el que requiere más trabajo, pero es con el que van a aprender más. 


Espero te  haya servido. Disculpa la tardanza en darte respuesta, estaba en clase.
